DevoxxClientMobile/src/main/java/com/devoxx/views/FeedbackPresenter.java

```diff
@@ -0,0 +1,129 @@
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.devoxx.views;
+
+import com.devoxx.DevoxxApplication;
+import com.devoxx.DevoxxView;
+import com.devoxx.model.Feedback;
+import com.devoxx.service.Service;
+import com.devoxx.util.DevoxxBundle;
+import com.gluonhq.charm.glisten.afterburner.GluonPresenter;
+import com.gluonhq.charm.glisten.control.AppBar;
+import com.gluonhq.charm.glisten.control.TextArea;
+import com.gluonhq.charm.glisten.control.TextField;
+import com.gluonhq.charm.glisten.control.TextInput;
+import com.gluonhq.charm.glisten.control.Toast;
+import com.gluonhq.charm.glisten.mvc.View;
+import com.gluonhq.charm.glisten.visual.MaterialDesignIcon;
+import com.gluonhq.cloudlink.client.user.User;
+import javafx.css.PseudoClass;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+
+import javax.inject.Inject;
+
+public class FeedbackPresenter extends GluonPresenter<DevoxxApplication> {
+
+    private static final PseudoClass PSEUDO_CLASS_ERROR = PseudoClass.getPseudoClass("error");
+
+    @FXML
+    private View feedback;
+
+    @FXML
+    private TextField name;
+    
+    @FXML
+    private TextField email;
+    
+    @FXML
+    private TextArea message;
+    
+    @Inject
+    private Service service;
+    
+    public void initialize() {
+        
+        feedback.setOnShowing( event -> {
+            final User authenticatedUser = service.getAuthenticatedUser();
+            if (authenticatedUser != null) {
+                name.setText(authenticatedUser.getName());
+                email.setText(authenticatedUser.getEmail());
+            }
+            
+            final Button close = MaterialDesignIcon.CLEAR.button(e -> DevoxxView.SESSIONS.switchView());
+            final Button send = MaterialDesignIcon.SEND.button(e -> {
+                final String nameText = name.getText();
+                final String emailText = email.getText();
+                final String messageText = message.getText();
+                if (validateInput(name, email, message)) {
+                    sendFeedback(nameText, emailText, messageText);
+                }
+            });
+            AppBar appBar = getApp().getAppBar();
+            appBar.setNavIcon(close);
+            appBar.setTitleText(DevoxxView.FEEDBACK.getTitle());
+            appBar.getActionItems().add(send);
+        });
+        
+        feedback.setOnShown(e -> feedback.requestFocus());
+        addValidator(name, email, message);
+    }
+
+    private void sendFeedback(String nameText, String emailText, String messageText) {
+        final Feedback feedback = new Feedback(nameText, emailText, messageText);
+        service.submitFeedback(feedback);
```

Comment:
After the feedback is sent or if you cancel the feedback view, the dialog on top of the content catalog is still showing. Shouldn't you remove it?

Reply:
Good catch!